### PR TITLE
sql: manual compact GA

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

 Manual Compact GA. Delete description about experimental feature for ALTER TABLE ... COMPACT

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]
LGTM

**Review by breezewish — APPROVED** [review]
Thanks a lot!

**Comment by breezewish** [comment]
I have also checked the cherry pick for v6.1. However the documentation may need some tunings for it, as it is experimental in v6.1.0, and GA in v6.1.1. Do you have some suggestions about some words for this branch?

File: sql-statements/sql-statement-alter-table-compact.md [hunk 1]
```diff
@@ -5,10 +5,6 @@ summary: TiDB 数据库中 ALTER TABLE ... COMPACT 语句的使用概况。
 
 # ALTER TABLE ... COMPACT
 
-> **警告：**
->
-> 该语句目前是实验性功能，不建议在生产环境中使用。
-
 TiDB 存储节点在后台会自动发起数据整理（Compaction）。数据整理时，表中的物理数据会被重写，如清理已删除的数据、合并多版本数据等，从而可以获得更高的访问性能，并减少磁盘空间占用。使用 `ALTER TABLE ... COMPACT` 语句可以立即对指定的表进行数据整理，而无需等待后台触发。
 
 该语句执行时不会阻塞现有 SQL 语句的执行或 TiDB 功能的使用，包括事务、DDL、GC 等，也不会改变通过 SQL 语句访问获得的数据内容。但执行过程中会占用一定量的 IO 及 CPU 资源，可能对业务延迟造成影响。
```
Comment (moyun) [inline sql-statements/sql-statement-alter-table-compact.md:10]: 建议加一句与上个版本实验功能异同的描述，提请用户注意。

Comment (moyun) [inline sql-statements/sql-statement-alter-table-compact.md:10]: > 但执行过程中会占用一定量的 IO 及 CPU 资源，可能对业务延迟造成影响。
这句话这么调整一下如何？ 
在系统自动数据整理间隙手动执行此 `COMPACT` 命令也会消耗一定量的 IO 及 CPU 资源（与自动方式类似），需要用户注意合适时机，选择资源空闲时段，避免对业务造成负面影响。

**Comment by breezewish** [comment]
Update: The syntax change will not be introduced to v6.1. I have removed the "cherry pick" check for v6.1.

File: sql-statements/sql-statement-alter-table-compact.md [hunk 2]
```diff
@@ -21,9 +17,11 @@ TiDB 存储节点在后台会自动发起数据整理（Compaction）。数据
 
 ```ebnf+diagram
 AlterTableCompactStmt ::=
-    'ALTER' 'TABLE' TableName 'COMPACT' 'TIFLASH' 'REPLICA'
+    'ALTER' 'TABLE' TableName 'COMPACT' ( 'TIFLASH' 'REPLICA' )?
 ```
 
+自 v6.2.0 起，语法中 `TIFLASH REPLICA` 部分可以被省略。
```
Comment (moyun) [inline sql-statements/sql-statement-alter-table-compact.md:23]: 部分可以被省略... 后面补充一小句： `含义相同，即默认只对 TiFlash 列存有效`。

File: sql-statements/sql-statement-alter-table-compact.md [hunk 3]
```diff
@@ -5,13 +5,9 @@ summary: TiDB 数据库中 ALTER TABLE ... COMPACT 语句的使用概况。
...
 # ALTER TABLE ... COMPACT
 
-> **警告：**
->
-> 该语句目前是实验性功能，不建议在生产环境中使用。
-
 TiDB 存储节点在后台会自动发起数据整理（Compaction）。数据整理时，表中的物理数据会被重写，如清理已删除的数据、合并多版本数据等，从而可以获得更高的访问性能，并减少磁盘空间占用。使用 `ALTER TABLE ... COMPACT` 语句可以立即对指定的表进行数据整理，而无需等待后台触发。
 
-该语句执行时不会阻塞现有 SQL 语句的执行或 TiDB 功能的使用，包括事务、DDL、GC 等，也不会改变通过 SQL 语句访问获得的数据内容。但执行过程中会占用一定量的 IO 及 CPU 资源，可能对业务延迟造成影响。
+该语句执行时不会阻塞现有 SQL 语句的执行或 TiDB 功能的使用，包括事务、DDL、GC 等，也不会改变通过 SQL 语句访问获得的数据内容。该语句执行时会消耗一定量的 IO 及 CPU 资源，需要用户注意合适时机，选择资源空闲时段，避免对业务造成负面影响。
```
Comment (hfxsd) [inline sql-statements/sql-statement-alter-table-compact.md:10]:
```suggestion
该语句执行时不会阻塞现有 SQL 语句的执行或 TiDB 功能的使用，包括事务、DDL、GC 等，也不会改变通过 SQL 语句访问获得的数据内容。该语句执行时会消耗一定量的 IO 及 CPU 资源，请注意选择合适的时机执行，如资源空闲时段，避免对业务造成负面影响。
```

**Pull request merged**